zerver/management/commands/export.py

```diff
@@ -144,8 +144,14 @@ def handle(self, *args: Any, **options: Any) -> None:
 
             print(f"\n\033[94mMessage content:\033[0m\n{message.content}\n")
 
-            user_count = UserProfile.objects.filter(realm_id=realm.id).count()
-            print(f"\033[94mNumber of users that reacted outbox:\033[0m {len(reactions)} / {user_count} total users\n")
+            user_count = UserProfile.objects.filter(
+                realm_id=realm.id,
+                is_active=True,
+                is_bot=False,
+            ).exclude(
+                role=UserProfile.ROLE_GUEST,
```

Comment:
Maybe add a comment noting that guest users can be exported, but generally aren't a priority for looking at whether most users are being exported.